### ISSUE TITLE
Shortcut issues with Firefox on Linux

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#2338]
Originally reported on Google Code with ID 1586

```
These issues were all observed using Firefox 22 with Flash 11.2 on Ubuntu linux. Some
of them might be specific to my keyboard layout; I'm using a US English keyboard.

control-shift-Minus  - Minimize current window
doesn't work on US keyboard (Note that typing control-Minus without shift worked)

control-shift-Plus - Maximize current window
doesn't work on US keyboard. I couldn't find any combination of keys that worked for
this function.
This function would normally be specified as "control-Plus", and the shift is implied
because typing + requires shift.

control-shift-Q - Focus out of the Flash window
no observable effect. What is this supposed to do?

control-shift-I - Focus the chat input field
This shortcut works, but after it focuses the input field, it types an "I" in the chat
field, which you have to backspace to remove

```

Reported by `calvin.walton@kepstin.ca` on 2013-07-15 18:49:08
